### PR TITLE
Ratings and Review: Dashboard Component Complete

### PR DESCRIPTION
**Pull request by eriknewland**
1. Fully responsive, additive, filter bar. Clicking on the bar representing a specific rating toggles reviews of that rating in the review-list feed.
2. Characteristic ratings using input sliders.
3. Extensive refactoring to pass linter guidelines.
4. Refactor of ReviewTile to include images as thumbnail, onclick creates modal experience. Clicking anywhere outside of image closes modal.
5. Have ~10 linter errors that will likely need exception. Majority are divs/images with special onClick handlers. My useEffects also seem to trigger a linter concern.

### CODE REVIEW

File: client/src/components/ratings/Ratings.jsx [hunk 1]
```diff
@@ -3,35 +3,45 @@ import fetcher from '../../fetchers';
...
   }, [feature]);
 
   return (
     <div>
       <div className="review-header">Ratings and Reviews</div>
       <div className="ratings-parent">
-        <ReviewDashboard reviews={reviews} setSelectedRating={setSelectedRating} selectedRating={selectedRating}/>
-        <ReviewList reviews={reviews} selectedRating={selectedRating}/>
+        <ReviewDashboard
+          reviews={reviews}
```
Comment (VesLan) [inline client/src/components/ratings/Ratings.jsx:38]: Probably remove the empty line here.

File: client/src/components/ratings/sub-components/DashboardCharacteristicTracker.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import CharacteristicBar from './CharacteristicBar.jsx';
+
+export default function CharacteristicTracker({ reviewMeta }) {
+  const charTrackerMapper = () => {
```
Comment (VesLan) [inline client/src/components/ratings/sub-components/DashboardCharacteristicTracker.jsx:5]: Although I understand, but feel like this format is a little bit confusing. Maybe refactor later.

File: client/src/components/ratings/sub-components/ReviewDashboard.jsx [hunk 3]
```diff
@@ -1,96 +1,99 @@
-import React, {useState, useEffect} from 'react';
-import ReviewTracker from './DashboardReviewTracker.jsx';
+import React, { useState, useEffect } from 'react';
```
Comment (VesLan) [inline client/src/components/ratings/sub-components/ReviewDashboard.jsx:1]: For this file, demo later.

File: client/src/components/ratings/sub-components/TrackerBar.jsx [hunk 4]
```diff
@@ -10,7 +9,7 @@ export default function TrackerBar ({progress}) {
     <div className="review-tracker-bar-container">
       <div className="review-tracker-bar-parent">
         <div className="review-tracker-bar-child" style={childDiv}>
-          <span className="review-tracker-bar-percent">{`${progress}%`}</span>
+          {/* <span className="review-tracker-bar-percent">{`${progress}%`}</span> */}
```
Comment (VesLan) [inline client/src/components/ratings/sub-components/TrackerBar.jsx:12]: Will you delete this?

**Review by VesLan — COMMENTED** [review]
Too much work and functionality, need demo later. Otherwise just read the comment and approve and merge.

File: client/src/components/ratings/sub-components/TrackerBar.jsx (hunk 4 above)
Comment (eriknewland) [reply client/src/components/ratings/sub-components/TrackerBar.jsx:12]: Got it!

File: client/src/components/ratings/sub-components/DashboardCharacteristicTracker.jsx (hunk 2 above)
Comment (eriknewland) [reply client/src/components/ratings/sub-components/DashboardCharacteristicTracker.jsx:5]: We can discuss during a demo

**Pull request closed without merging**